xdsl/ir/affine/affine_map.py

```diff
@@ -15,6 +15,18 @@ class AffineMap:
     num_symbols: int
     results: list[AffineExpr]
 
+    @staticmethod
+    def from_constant(value: int) -> AffineMap:
+        return AffineMap(0, 0, [AffineExpr.constant(value)])
```

Comment:
Can we call this constant_map?

Reply:
How about just `constant`?

Reply:
I prefer `constant_map`. It's also what mlir uses.